### DISCUSSION TITLE
OpenSnitch blocking mounted network shares

### DISCUSSION THREAD

**Discussion by wsbregard** [evilsocket/opensnitch#1181] [Q&A]
Hi all, I'm using Linux Mint 22 Cinnamon and have several network shares configured to mount at boot using /etc/fstab. When OpenSnitch is disabled I can access them fine, but when I enable OpenSnitch it blocks access without any prompts to allow access. How can I configure OpenSnitch to allow access to my mounted network shares?

**Comment by gustavo-iniguez-goya** [comment]
What version of opensnitch do you have installed? Be sure that you're using ebpf as `Process monitor method` (Preferences -> Nodes) , and that it works (kernel modules should be under `/usr/lib/opensnitchd/ebpf/`

also you can verify if your kernel has all the prerequisities with: `opensnitchd -check-requirements`

**Reply by DavParc** [reply]
I have the same problem, unfortunately. BTW, the 'opensnitchd -check-requirements' is not recognised on my system (flag provided but not defined). I'm on Mint 22 kernel 6.8.71.

**Reply by DavParc** [reply]
Oh, and opensnitch v1.5.8

**Reply by gustavo-iniguez-goya** [reply]
@DavParc , upgrade to the v1.7.x or v1.6.x versions if possible. Also verify that you have the ebpf modules under /etc/opensnitchd/ (opensnitch.o) and review the log /var/log/opensnitchd.log. There should be errors if the modules couldn't be loaded, which is the typical reason for mounted shares not to work.

On the other hand, you can enable `[x] Intercept unknown connections` from the Preferences dialog. You'll be prompted to allow an "Outgoind connection", which you can use to allow by destination IP + port.

**Comment by DavParc** [comment]
Thanks Gustavo for your quick reply and a great app.
Updating did the trick!
Cheers
Dave
